src/module-5/pagination/index.js

```diff
@@ -2,30 +2,135 @@ export default class Pagination {
   element;
   start = 0;
   pageIndex = 0;
+  subElements = {};
+
+  goToPrevPage = (event) => {
+    event.preventDefault();
+    this.page -= 1;
```

Comment:
if it's possible, try to operate page index only to avoid confusion between these two values;  
you may have a view (getter), like `pageLabel` or `pageCount` which will be `this.pageIndex + 1`